src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -264,7 +264,8 @@ internal enum WellKnownType
 
         Microsoft_CodeAnalysis_Runtime_Instrumentation,
 
-        System_Runtime_CompilerServices_ReadOnlyAttribute,
+        System_Runtime_CompilerServices_IsReadOnlyAttribute,
+        Microsoft_CodeAnalysis_EmbeddedAttribute,
```

Comment:
>Microsoft_CodeAnalysis_EmbeddedAttribute [](start = 8, length = 40)

I am just started looking at this change, but the fact that we added this type to the well-known types list feels very surprising to me. Well-known types are not used to identify attribute applications, and, when we are embedding types we are using the types that we embed, which aren't well-known (they are hidden from everyone, but the compiler).